cassandra-migration/src/main/java/org/cognitor/cassandra/migration/MigrationTask.java

```diff
@@ -29,8 +30,19 @@
      * @param repository the repository that contains the migration scripts
      */
     public MigrationTask(Database database, MigrationRepository repository) {
+        this(database, repository, false);
+    }
+
+    /**
+     * Creates a migration task that uses the given database and repository.
+     *
+     * @param database the database that should be migrated
+     * @param repository the repository that contains the migration scripts
+     */
+    public MigrationTask(Database database, MigrationRepository repository, boolean withConsensus) {
```

Comment:
please add the @param documentation for "withConsensus"

Reply:
good catch